pep-0585.rst

```diff
@@ -112,6 +112,9 @@ Python 3.9, the following collections become generic using
 * ``collections.abc.ValuesView``
 * ``contextlib.AbstractContextManager``  # typing.ContextManager
 * ``contextlib.AbstractAsyncContextManager``  # typing.AsyncContextManager
+* ``re.Pattern``  # typing.Pattern, typing.re.Pattern
+* ``re.Match``  # typing.Match, typing.re.Match
+* ``io.IO``  # typing.IO, typing.io.IO
 
```

Comment:
I believe there are still other containers that should be made generic like `queue.*Queue`, should the PEP enumerate all of them?

Reply:
Honestly I think the PEP should stick to things that are currently shadowed in typing.py. Once we have GenericAlias it's easy enough to make other classes generic.

Reply:
Okay sounds good!